python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-pim-sparse-mode.j2

```diff
@@ -59,6 +66,13 @@ router pim sparse-mode
 {%             if vrf.ipv4.bfd is arista.avd.defined(true) %}
          bfd
 {%             endif %}
+{%             if vrf.ipv4.make_before_break is arista.avd.defined %}
+{%                 if vrf.ipv4.make_before_break: %}
+         make-before-break
+{%                 else %}
+         make-before-break disabled
+{%                 endif %}
+{%             endif %}
```

Comment:
```suggestion
{%             if vrf.ipv4.make_before_break is arista.avd.defined(true) %}
         make-before-break
{%             elif vrf.ipv4.make_before_break is arista.avd.defined(false) %}
         make-before-break disabled
{%             endif %}
```

Reply:
David did as I asked because that implementation will only result in one `if` for most deployments (very few will have this setting), where the other if/elif will make two rounds every time.

Reply:
colons removed

Reply:
hmm ok I need to revert back I just read this .. apologies will take care of this!